source/common/statistic_impl.cc

```diff
@@ -236,4 +236,42 @@ nighthawk::client::Statistic HdrStatistic::toProto(SerializationDomain domain) c
   return proto;
 }
 
-} // namespace Nighthawk
\ No newline at end of file
+StatisticPtr CircllhistStatistic::combine(const Statistic& statistic) const {
+  auto combined = std::make_unique<CircllhistStatistic>();
+  const auto& b = dynamic_cast<const CircllhistStatistic&>(statistic);
```

Comment:
Let's find a better (more expressive) variable name than just "b".

Reply:
changed it to "stat".